src/renderer/components/configure-recording-set-page/configure-recording-set-page.test.tsx

```diff
@@ -59,12 +112,130 @@ describe('ConfigureRecordingSetPage', () => {
       voiceDvcfg: 'voiceDvcfg',
     });
 
-    render(<ConfigureRecordingSetPage onSettingsButtonClick={noop} onNext={noop} onBack={noop} onSetSelected={noop} />);
+    render(
+      <ConfigureRecordingSetPage
+        onSettingsButtonClick={noop}
+        onNext={noop}
+        onBack={noop}
+        onSetSelected={noop}
+        openFilePicker={mockedFilePicker}
+      />,
+    );
+
     userEvent.selectOptions(screen.getByTestId('selectReclist'), 'デルタ式英語リストver5 (Delta English Ver. 5)');
     userEvent.click(await screen.findByTestId('show-details-button'));
 
     await screen.findByText('listContent');
 
     expect(recordingListDataService.readData).toHaveBeenCalledTimes(1);
+
+    userEvent.click(await screen.findByTestId('up-button'));
```

Comment:
Why is this needed?

Reply:
For some reason, the state wasn't resetting between tests, and I couldn't find a cleanup function that would reset it, so I had to manually go back to the main page between tests

Reply:
Have you tried `cleanup()` from react-testing-library?

Reply:
Yes, it wasn't actually resetting the state